tests/boulder-integration.sh

```diff
@@ -430,6 +430,12 @@ for path in $archive $conf $live; do
     fi
 done
 
+# Test ACMEv2-only features
+if [ "${BOULDER_INTEGRATION:-v1}" = "v2" ]; then
+    common -a manual -d *.le4.wtf,le4.wtf --preferred-challenges dns \
```

Comment:
I would suggest quoting the asterisk here (even though it will expand to an asterisk if there are no matching files, and there will probably never be any matching files)